modules/docs/Makefile

```diff
@@ -0,0 +1,21 @@
+GOMPLATE_VERSION ?= v2.2.0
+GOMPLATE_REPO ?= hairyhenderson/gomplate
+GOMPLATE ?= gomplate
+
+## Install dependencies
+docs\:deps:
+	@make github:download-public-release \
+		REPO=$(GOMPLATE_REPO) \
+		FILE=$(GOMPLATE)_$(OS)-amd64 \
+		VERSION=$(GOMPLATE_VERSION) \
+		OUTPUT=/usr/local/bin/$(GOMPLATE) > /dev/null
```

Comment:
This should install to a path relative to the `build-harness` 
a) avoids permission issues
b) avoids conflicts between different `build-harness` checkouts